app/views/pages/resources.html.erb

```diff
@@ -1,9 +1,100 @@
 <%- @page_title = 'Tools & Resources' -%>
 
-<div class="usa-section main-section">
+<div class="usa-section main-section tools-resources">
   <h1><%= @page_title %></h1>
+  <div class="usa-grid tools-links">
+    <ul class="usa-unstyled-list usa-width-one-half tools-list">
+      <li>
+        <%= link_to weight_estimator_path do %>
+          <%= image_tag 'icons/weight-estimator.svg' %>
```

Comment:
If this (and the other icons) infer no meaning and are purely decoration, add `alt: '', role: 'presentation'` here.

```erb
<%= image_tag 'icons/weight-estimator.svg', alt: '', role: 'presentation' %>
```

The `role="presentation"` attribute and value will prevent screen readers from announcing the image file name or whatever Rails defaults to. Without the `role` attribute, this could be read aloud as "Weight Estimator Household Goods Weight Estimator."